packages/utils/schema/user-profile.graphql

```diff
@@ -0,0 +1,111 @@
+type ProfileName
+  @createModel(accountRelation: SINGLE, description: "Your name") {
+  name: String! @string(maxLength: 150)
+}
+
+type ProfileAvatar
+  @createModel(
+    accountRelation: SINGLE
+    description: "An image and optional fallback images representing your avatar"
+  ) {
+  original: ImageMetadata!
+  alternatives: [ImageMetadata] @list(maxLength: 3)
```

Comment:
The structure I would like to have is a set of `Dimension` descriptions with an associated list of `Files`.

The dimension description will include `width` & `height` and `filesize`. Also it will include an aspect ratio. Raster images would put in their width & height, and vectors would use `aspectRatio` to  represent they can be flawlessly resized to any dimensions.

```suggestion
  sizes: [Dimensions] @list(maxLength: 69420) → Files
```

Reply:
This schema was just taken from the original basic profile data model. I think it's a bit more ergonomic than your proposal, as one can just ignore the alternative images if they don't care about them